README.md

```diff
@@ -11,6 +11,103 @@ go get github.com/moolitayer/golang/glog # Dependency
 go get github.com/moolitayer/awx-client-go/awx
 ```
 
+## Usage
+### import
+```
+import 	"github.com/moolitayer/awx-client-go/awx"
+```
+
+### Creating a connection:
+```
+# Uses the builder pattern:
+connection, err := awx.NewConnectionBuilder().
+  Url("http://awx.example.com/api").
+  Username(username).
+  Password(password).
+  Token("TOKEN").
+  Bearer("BEARER").
+  CAFile("/etc/pki/tls/cert.pem").
+  Insecure(insecure).
+  Proxy(http://myproxy.example.com).
+  Build()                                      # Create the client
+if err != nil {
+  panic(err)
+}
+defer connection.Close()                      # Don't forget to close the connection!
+```
+`Url()` points at an AWX server's root API endpoint.
+`Proxy()` specifies A proxy server to use for all outgoing connection to the AWX server.
+#### Authentication
+`Username()` and `Password()` specify Basic Auth for AWX API server.
+`Token()` uses the authtoken/ endpoint and works with AWX XXX and Ansible tower < 3.3.
```

Comment:
"AWX XXX"? If we don't know the version number it's probably best to reword this to something like "works with older AWX versions"

Reply:
I'll update, should be 1.0.5 https://groups.google.com/forum/#!searchin/awx-project/release%7Csort:date/awx-project/K80oF64C5EQ/1hWCKY5SAAAJ